resources/views/Widgets/Common/BackgroundWidget.twig

```diff
@@ -29,6 +30,7 @@
     {{ Twig.set("url", "category.details[0].image2Path") }}
 {% elseif sourceType == "custom-image" %}
     {{ Twig.set("url", customImagePath | json_encode) }}
+    {{ Twig.set("fallbackUrl", fallbackImagePath | json_encode) }}
```

Comment:
Warum setzt du hier Wert nochmal per TwigBuilder in eine Variable? Der Wert sollte sich nicht mehr während des Seitenaufrufs ändern. Stattdessen könntest du den Wert aus den Widget-Settings direkt an die Vue-Komponente übergeben.